### DISCUSSION TITLE
Why doesn't archive work?

### DISCUSSION THREAD

**Discussion by acceleratxr** [ericaltendorf/plotman#388] [Q&A] [answered]
I have my rsync set up correctly. I can manually run an rsync command no problem. However, when in plotman interactive I see it attempting to run an rsync to the archive server it doesn't actually do anything.

For example here is my current plotman screen.
![image](https://user-images.githubusercontent.com/43181229/118065861-8da4a580-b352-11eb-8292-66031c528730.png)

Yet top shows no `rsync` process running. If I perform an `ls` on my dst directory that plotman is supposedly syncing from I do not found the file listed in plotman, instead there is another file. I presume this is why it's not working but doesn't really explain why it isn't syncing the actual file?

```
ubuntu@chia:~$ ls -l /chia/plot-tmp
total 106248192
drwx------ 2 root   root          16384 May 11 23:02 lost+found
-rw-rw-r-- 1 ubuntu ubuntu 108798118036 May 12 18:05 plot-k32-2021-05-12-06-59-dca71ee71f1874e1d195b20b8589c5e54a7ea0d05001c1dd8bc4ea5d6ca56b3c.plot
ubuntu@chia:~$
```

Is there a bug? or do I have something set up wrong?

**Comment by altendky** [comment, marked as answer]
Try the exact command that plotman shows in it's on-screen log and also check the log from the rsyncd server with something like `journalctl -fu rsync.service`.

**Reply by acceleratxr** [reply]
The exact command doesn't work because as noted above the file referenced doesn't exist. It's another plot file that's in the dst directory.

**Reply by altendky** [reply]
So sure, change the file name but leave every other bit the same.  The point is to make it happen in a situation where you get to see the error from one or both ends.  And to be doing that with as close as possible to the plotman command.

**Reply by acceleratxr** [reply]
When I do that it transfers no problem.

**Reply by acceleratxr** [reply]
OK I take that back. I wasn't seeing the entire command being print out correctly. The command as output by plotman doesn't work. The reason is because it is appending a port number to the remote server's URL which is invalid. It turns out I hadn't configured the port to 12000 on the rsync server as the wiki's example config suggests.

**Reply by altendky** [reply]
Presently you must use 12000 or edit the source.  We are working on a generalization that makes this configurable.

**Comment by cojarbi** [comment]
@acceleratxr did you manage to solve this? Im having the same issue, 

Journal says
May 31 14:27:52 RyzenUbuntu systemd[1]: Condition check resulted in fast remote file copy program daemon being skipped.
-- Reboot --
May 31 14:30:08 RyzenUbuntu systemd[1]: Condition check resulted in fast remote file copy program daemon being skipped.
-- Reboot --
May 31 15:46:25 RyzenUbuntu systemd[1]: Condition check resulted in fast remote file copy program daemon being skipped.
-- Reboot --
May 31 16:19:05 RyzenUbuntu systemd[1]: Condition check resulted in fast remote file copy program daemon being skipped.
-- Reboot --
May 31 17:37:50 RyzenUbuntu systemd[1]: Condition check resulted in fast remote file copy program daemon being skipped.
May 31 19:36:42 RyzenUbuntu systemd[1]: Started fast remote file copy program daemon.
May 31 19:42:52 RyzenUbuntu systemd[1]: Stopping fast remote file copy program daemon...
May 31 19:42:53 RyzenUbuntu systemd[1]: rsync.service: Succeeded.
May 31 19:42:53 RyzenUbuntu systemd[1]: Stopped fast remote file copy program daemon.
May 31 19:42:53 RyzenUbuntu systemd[1]: Started fast remote file copy program daemon.

**Reply by altendky** [reply]
https://github.com/ericaltendorf/plotman/discussions/388#discussioncomment-732302

**Reply by cojarbi** [reply]
@altendky i think my port is listening on the server

COMMAND   PID USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
rsync   11913 root    4u  IPv4 159023      0t0  TCP *:12000 (LISTEN)
rsync   11913 root    5u  IPv6 159024      0t0  TCP *:12000 (LISTEN)

**Comment by cojarbi** [comment]
Ok its seems to be working now, was a path issue which was @acceleratxr issue a day earlier of him posting this

Thanks @altendky
